apps/prairielearn/src/ee/auth/lti13/lti13Auth.html.ts

```diff
@@ -47,6 +47,43 @@ export function Lti13Test({
   `.toString();
 }
 
+export function Lti13AuthIframe({ parameters }: { parameters: Record<string, any> }) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${HeadContents({ resLocals: {}, pageTitle: 'LTI 1.3 redirect' })}
+      </head>
+      <body>
+        <main id="content" class="m-3">
+          <form id="interceptForm" method="POST" action="" target="_blank">
+            ${Object.entries(parameters).map(
+              ([key, value]) =>
+                html`<input type="hidden" name="${key}" value="${String(value)}" />`,
+            )}
+            <button id="submitButton" class="btn btn-primary btn-lg">
```

Comment:
```suggestion
            <button id="submitButton" class="btn btn-primary mb-2">
```

IMO this looks a little nicer.

<img width="850" alt="Screenshot 2025-04-30 at 11 31 53" src="https://github.com/user-attachments/assets/808b0f02-999e-4a9d-9416-26371f0afd3a" />